src/renderer/components/wizard/wizard.scss

```diff
@@ -15,10 +15,14 @@
   }
 
   @mixin scrollableContent() {
-    @include custom-scrollbar($theme: dark);
+    overflow: auto;
     padding: var(--wizard-spacing);
     height: var(--wizard-content-height);
     max-height: var(--wizard-content-max-height);
+
+    &::-webkit-scrollbar-thumb {
+      background-color: #bbbbbb; // Always light scrollbar in dialogs
```

Comment:
Do we actually want to force this or should we make it a variable like all other colours?

Reply:
Removed these lines.